frontend/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Flex, Box, Heading, Spacer, ButtonGroup } from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+
+const Header = () => {
+	return (
+		<Flex minWidth="max-content" alignItems="center" bg="blue.300" gap={2} p={2}>
+			<Box p={2}>
+				<Heading size="md" colorScheme="white">Self E-Learning System</Heading>
+			</Box>
+			<Spacer />
+			<ButtonGroup gap={2}>
+				<Link to="">Categories</Link>
+				<Link to="">Profile</Link>
```

Comment:
Add `#` as href for temporary link to make sure its a link jd. 
```suggestion
				<Link to="#">Categories</Link>
				<Link to="#">Profile</Link>
```